Classes/Command/ProjectionCommandController.php

```diff
@@ -173,6 +180,32 @@ public function catchUpCommand($projection)
     }
 
     /**
+     * Listen to new events for a given (asynchronous) projection
+     *
+     * @param string $projection The projection identifier; see projection:list for possible options
+     * @param int $lookupInterval Pause between lookups (in seconds)
+     * @return void
+     * @see neos.cqrs:projection:list
+     * @see neos.cqrs:projection:catchup
+     */
+    public function watchCommand($projection, $lookupInterval = 10)
+    {
+        try {
+            $projectionDto = $this->projectionManager->getProjection($projection);
+        } catch (\InvalidArgumentException $e) {
+            $this->outputLine('<error>%s</error>', [$e->getMessage()]);
+            $this->quit(1);
+            return;
+        }
+
+        $this->outputLine('Watching events for projection "%s" ...', [$projectionDto->getIdentifier()]);
+        do {
+            Scripts::executeCommand('neos.cqrs:projection:catchup', $this->flowSettings, false, ['projection' => $projectionDto->getIdentifier()]);
```

Comment:
When just calling `ProjectionManager::catchUp()` `persistAll()` wouldn't be called. Besides this process would need more and more memory..